src/mame/drivers/cischeat.cpp

```diff
@@ -1267,17 +1272,17 @@ static INPUT_PORTS_START( cischeat )
 	PORT_DIPUNKNOWN_DIPLOC( 0x40, 0x40, "SW301:7" )
 	PORT_DIPUNKNOWN_DIPLOC( 0x80, 0x80, "SW301:8" )
 	// DSW 2
-	PORT_DIPNAME( 0x0300, 0x0300, "Unit ID" ) PORT_DIPLOCATION("SW302:1,2") // -> !f0020 (ID of this unit, when linked)
-	PORT_DIPSETTING(      0x0300, "0 (Red Car)" )
-	PORT_DIPSETTING(      0x0200, "1 (Blue Car)" )
-	PORT_DIPSETTING(      0x0100, "2 (Yellow Car)" )
-	PORT_DIPSETTING(      0x0000, "3 (Green Car)" )
+	PORT_DIPNAME( 0x0300, 0x0300, "Unit ID" ) PORT_DIPLOCATION("SW302:2,1") // -> !f0020 (ID of this unit, when linked)
```

Comment:
This really doesn’t make sense – without a schematic and board shot showing the connections reversed for these two bits, I find it pretty hard to believe that the DIP switches map to bits in ascending order except for these two that are reversed.